app/api/groups/get_group.feature

```diff
@@ -82,6 +87,9 @@ Feature: Get group by groupID (groupView)
       "code_expires_at": "2017-10-13T05:39:48Z",
       "open_activity_when_joining": true,
       "current_user_is_manager": true,
+      "can_manage": "memberships_and_group",
+      "can_grant_group_access": true,
+      "can_watch_members": true,
```

Comment:
actually, shouldn't we call these attributes `current_user_can_...` to be consistent with "current_user_is_manager" and as it is not a group attribute?

Reply:
fixed